projects/demo/src/modules/components/tag/examples/4/index.html

```diff
@@ -0,0 +1,16 @@
+<ng-container *ngFor="let tag of tags; let index = index;">
+    <tui-tag
+        size="l"
+        status="custom"
+        class="tag"
+        [value]="tag"
+        [hoverable]="true"
+        [editable]="true"
+        [removable]="true"
+        (edited)="handleTagEdited($event, index)"
+    ></tui-tag>
+</ng-container>
+
+<div *ngIf="tags.length === 0">
```

Comment:
- Don't know why layout might break?

![image](https://user-images.githubusercontent.com/12021443/150952411-5f88552b-34bc-42f1-8964-9a3e821d7007.png)

- It would be great to be able to add tags if all have been removed

![image](https://user-images.githubusercontent.com/12021443/150952509-de5ac39f-881c-4676-8bae-ca4b6f615d45.png)


maybe as example, @waterplea what do you think?

![image](https://user-images.githubusercontent.com/12021443/150952779-82028355-40c6-4f6b-964b-a60536e41aa2.png)

Reply:
I don't experience this:
<img width="588" alt="Screenshot 2022-01-26 at 12 18 08" src="https://user-images.githubusercontent.com/17404191/151119669-b8ed883d-de07-4c4a-9408-cfdd74107957.png">